stable/elasticsearch/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: elasticsearch
 home: https://www.elastic.co/products/elasticsearch
-version: 1.11.1
```

Comment:
Please update the version.
A PR was merged that has the same version number.
https://github.com/helm/charts/blob/master/stable/elasticsearch/Chart.yaml#L3

Reply:
Done!